torchci/components/common/LabelGuide.tsx

```diff
@@ -0,0 +1,66 @@
+import { styled, SxProps, Theme } from "@mui/material/styles";
+import React from "react";
+
+// Define the styles for label box component
+const Item = styled("div")(({ theme }) => ({
+  display: "flex",
+  justifyContent: "flex-start",
+  flexWrap: "wrap",
+}));
+
+const LabelItem = styled("div")(({}) => ({
+  display: "flex",
+  flexDirection: "row",
+  alignItems: "center",
+  justifyContent: "space-between",
+}));
+
+const LabelIcon = styled("div")(({}) => ({
+  fontSize: "15px",
+  marginRight: "6px",
+  marginLeft: "10px",
+  height: "auto",
+}));
+
+const LabelBoxTitle = styled("div")(({}) => ({
+  marginBottom: "5px",
+}));
+
+/**
+ * Props for LabelList component
+ */
+export interface LabelInfo {
+  name: string;
+  type: string;
+  render: (className?: string) => JSX.Element;
+}
+
+export function LabelGuide({
+  title,
+  props,
+  labelInfoList,
+  labelStyleList,
+}: {
+  title?: string;
+  props?: SxProps<Theme>;
+  labelInfoList: LabelInfo[];
+  labelStyleList?: { [key: string]: string };
+}) {
+  return (
+    <Item sx={props}>
+      <LabelBoxTitle>{title ? `${title} :` : ""}</LabelBoxTitle>
+      {labelInfoList.map((labelInfo, index) => (
+        <LabelItem key={index}>
+          <LabelIcon key={index}>
+            {" "}
+            {labelInfo.render(
+              labelStyleList ? labelStyleList[labelInfo.type] : undefined
+            )}
+          </LabelIcon>
+          <div>{labelInfo.name}</div>
+        </LabelItem>
+      ))}
+    </Item>
+  );
+}
+export default LabelGuide;
```

Comment:
If you decide to keep the guide, I think it would be better to have things horizontally aligned?  Like the all null ~ and the not exist ~ should both be on the same row.  Idk how hard that would be tho
![image](https://github.com/user-attachments/assets/194e8766-362f-4c14-811f-1f7a0ded3502)

Reply:
IMO, I don't think we really need a legend table as most of the icons are easy to recognized.  Even if there is one, I would prefer it to be collapsed by default.  I think people will be confused about too much details like all null, not exist, neutral, warning only.

Reply:
Also, the label is already part of the job/group description when you hover the mouse over, so the legend table becomes redundant

Reply:
Sounds good! I can remove the label guide

Reply:
Can we still have a blinking icon for job that is in progress? 
Also, as both in-progress and queue are represented by similar icon,I wonder if gray clock could indicate that jobs is queued, while yellow blinking one means that it's in progress...

Reply:
I don't recall ever having a blinking icon for in progress jobs?

Reply:
@malfet  you mean like this lol? (Dark mode)
![shiningPending](https://github.com/user-attachments/assets/0c0fe7ab-ee4e-4d80-a8a5-bc6585af9518)